frontend/src/features/entity_selector/EntitySelector.tsx

```diff
@@ -283,102 +349,114 @@ const EntitySelectorInnerAuth = ({
           </Grid>
         </Box>
       )}
-      <>
-        {rating ? (
-          <EntityCard
-            compact
-            result={rating}
-            showRatingControl={showRatingControl}
-            onRatingChange={handleRatingUpdate}
-          ></EntityCard>
-        ) : (
-          <>
-            {(loading ||
-              !showEntityInput ||
-              entityAvailability === ENTITY_AVAILABILITY.UNAVAILABLE) && (
-              <EmptyEntityCard compact loading={loading} />
-            )}
-            {!loading &&
-              entityAvailability === ENTITY_AVAILABILITY.UNAVAILABLE && (
-                <Box
-                  display="flex"
-                  justifyContent="center"
-                  alignItems="center"
-                  position="absolute"
-                  top="0"
-                  color="white"
-                  bgcolor="rgba(0,0,0,.6)"
-                  width="100%"
-                  sx={{
-                    aspectRatio: '16/9',
-                    [theme.breakpoints.down('sm')]: {
-                      fontSize: '0.8rem',
-                    },
-                  }}
-                >
-                  <Typography textAlign="center" fontSize="inherit">
-                    {pollName === YOUTUBE_POLL_NAME
-                      ? t('entitySelector.youtubeVideoUnavailable')
-                      : t('entityCard.thisElementIsNotAvailable')}
-                  </Typography>
-                </Box>
+      <Slide
+        mountOnEnter
+        in={slideIn}
+        direction={slideDirection}
+        onEntered={onSlideEntered}
+        onExited={onSlideExited}
+        timeout={ENTITY_CARD_SWIPE_TIMEOUT}
+        appear={variant === 'noControl' ? false : true}
+      >
+        {/* position: relative is required to correctly display the entity unavailable box */}
+        <Box {...bindDrag()} sx={{ touchAction: 'pan-x' }} position="relative">
```

Comment:
For some reason, `pan-x` does not behave as expected on iOS 17.4 and cause all vertical gestures to be ignored:
```suggestion
        <Box {...bindDrag()} sx={{ touchAction: 'none' }} position="relative">
```